tests/libc/enc/enc.c

```diff
@@ -16,8 +16,37 @@
 #include "libc_t.h"
 #include "mtest.h"
 
+#pragma STDC FENV_ACCESS ON
+
+/* Type of the control word.  */
+typedef unsigned int fpu_control_t __attribute__((__mode__(__HI__)));
+/* Macros for accessing the hardware control word.  */
+#define _FPU_GETCW(cw) __asm__ __volatile__("fnstcw %0" : "=m"(*&cw))
+#define _FPU_SETCW(cw) __asm__ __volatile__("fldcw %0" : : "m"(*&cw))
+
 int t_status = 0;
 
+int my_printfpu_control()
+{
+    fpu_control_t cw;
+    _FPU_GETCW(cw);
+    // printf("FPU Control Word is 0x%x\n", cw);
+}
```

Comment:
I'm surprised that this compiled; return statement missin.g

Reply:
I am not calling the routine and so probably get a warning. Will fix or remove.

Reply:
Fixed